balta/src/main/scala/za/co/absa/db/balta/DBTestSuite.scala

```diff
@@ -44,7 +48,9 @@ abstract class DBTestSuite extends AnyFunSuite {
   /**
    * This is the connection info for the DB. It can be overridden in the derived classes to provide specific credentials
    */
-  protected lazy val connectionInfo: ConnectionInfo = readConnectionInfoFromConfig
+  protected lazy val connectionInfo: ConnectionInfo = {
```

Comment:
By this change, you kept the private method `readConnectionInfoFromConfig` without usage. Is that intention?

Reply:
Thanks for noticing!